llumnix/config/config.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024, Alibaba Group;
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List
+import yaml
+from yacs.config import CfgNode
+
+BASE_KEY = "_BASE_"
+
+class LlumnixConfig(CfgNode):
```

Comment:
We have `llumnix/llumnix_config.py`, `llumnix/config/config.py`, and even a `LlumnixConfig` class in `llumnix/config/config.py`. This looks confusing. Please clarify the naming of these directories / files / classes

Reply:
yes, this is very confusing ~!

llumnix/llumnix_config.py contains internal config structures (MigrationConfig and GlobalSchedulerConfig). Rename llumnix/llumnix_config.py -> llumnix/internal_config.py.

llumnix/config folder contains files about llumnix config parameters. rename llumnix/config/config.py ->  llumnix/config/llumnix_config.py.  And to make it concise, import LlumnixConfig in llumnix config package (see llumnix/config/__init__.py). So, just need " from llumnix.config import LlumnixConfig", not "from llumnix.config.llumnix_config import LlumnixConfig"